hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestContainerBalancerOperations.java

```diff
@@ -115,4 +125,47 @@ public void testContainerBalancerCLIOperations() throws Exception {
   }
 
   //TODO: add more acceptance after container balancer is fully completed
+
+  /**
+   * Test if Container Balancer CLI options override the default configurations.
+   */
+  @Test
+  public void testIfCBCLIOverridesConfigs() throws Exception {
+    boolean running = containerBalancerClient.getContainerBalancerStatus();
+    assertFalse(running);
+
+    //CLI option for threshold is not passed
+    Optional<Double> threshold = Optional.empty();
+
+    //CLI options are passed
+    Optional<Integer> maxDatanodesPercentageToInvolvePerIteration =
+            Optional.of(100);
+    Optional<Long> maxSizeToMovePerIterationInGB = Optional.of(1L);
+    Optional<Long> maxSizeEnteringTargetInGB = Optional.of(6L);
+    Optional<Long> maxSizeLeavingSourceInGB = Optional.of(6L);
+    Optional<Integer> iterations = Optional.of(10000);
+    Optional<Long> moveTimeout = Optional.of(65L);
+    Optional<Long> moveReplicationTimeout = Optional.of(55L);
+    Optional<Long> balancingInterval = Optional.of(70L);
+    Optional<Boolean> networkTopologyEnable = Optional.of(true);
+    Optional<String> includeNodes = Optional.of("");
+    Optional<String> excludeNodes = Optional.of("");
+    containerBalancerClient.startContainerBalancer(threshold, iterations,
+            maxDatanodesPercentageToInvolvePerIteration,
+            maxSizeToMovePerIterationInGB, maxSizeEnteringTargetInGB,
+            maxSizeLeavingSourceInGB, balancingInterval, moveTimeout,
+            moveReplicationTimeout, networkTopologyEnable, includeNodes,
+            excludeNodes);
+    running = containerBalancerClient.getContainerBalancerStatus();
+    assertTrue(running);
+
+    ContainerBalancerConfiguration config = cluster.getStorageContainerManager().getContainerBalancer().getConfig();
+
+    //If CLI option is not passed, it takes the default configuration
+    assertEquals(config.getThreshold(), 10);
+
+    //If CLI option is passed, it overrides the default configuration
+    assertEquals(config.getIterations(), 10000);
```

Comment:
Similar to the previous comment.
```suggestion
    assertEquals(10000, config.getIterations());
```